src/driver-sinowealth.c

```diff
@@ -0,0 +1,475 @@
+/*
+ * Copyright © 2020 Marian Beermann
```

Comment:
Not sure if I like it but I guess it's fine (IMHO if you contribute of a FOSS project you might as well give them the copyright of your code, but I understand that not everyone sees it that way).

Reply:
We have no CLA, if they write it, it's their copyright, but they're licensing it under MIT.

Reply:
> We have no CLA, if they write it, it's their copyright, but they're licensing it under MIT.

I know that it isn't necessary. I still think it would be nice, it doesn't matter if you install it from source, but for example in Debian you have to mention _every_ copyright notice that's in the code if you want it in the official archive. Depending on the project, that list can grow pretty quickly and it gets hard to manage it.